nipype/interfaces/ants/registration.py

```diff
@@ -395,6 +397,8 @@ class RegistrationOutputSpec(TraitedSpec):
     warped_image = File(desc="Outputs warped image")
     inverse_warped_image = File(desc="Outputs the inverse of the warped image")
     save_state = File(desc="The saved registration state to be restored")
+    metric_value = traits.Float(desc='the final value of metric')
+    elapsed_time = traits.Float(desc='the total elapsed time as reported by ANTs')
```

Comment:
Is there a difference between this `elapsed_time` and `duration` stored in every `CommandLineInterface` `runtime`? If you need this as an output it might be better to enable this for all interfaces.